examples/Google.Cloud.Functions.Examples.FSharpEventFunction/Function.fs

```diff
@@ -40,14 +40,18 @@ type Function() =
             printfn "Storage object information:"
             printfn "  Name: %s" data.Name
             printfn "  Bucket: %s" data.Bucket
-            printfn "  Size: %A" data.Size
+            printfn "  Size: %i" data.Size
             printfn "  Content type: %s" data.ContentType
             printfn "Cloud event information:"
             printfn "  ID: %s" cloudEvent.Id
-            printfn "  Source: %A" cloudEvent.Source
+            printfn "  Source: %O" cloudEvent.Source
             printfn "  Type: %s" cloudEvent.Type
             printfn "  Subject: %s" cloudEvent.Subject
-            printfn "  DataSchema: %A" cloudEvent.DataSchema
-            printfn "  DataContentType: %A" cloudEvent.DataContentType
-            printfn "  SpecVersion: %A" cloudEvent.SpecVersion
-            Task.CompletedTask
+            printfn "  DataSchema: %O" cloudEvent.DataSchema
+            printfn "  DataContentType: %O" cloudEvent.DataContentType
+            printfn "  Time: %s" (cloudEvent.Time |> Option.ofNullable |> Option.map(fun time -> time.ToUniversalTime().ToString "yyyy-MM-dd'T'HH:mm:ss.fff'Z'") |> Option.defaultValue "")
```

Comment:
This definitely brings it into parity with the C# example... but do you think it's worth the extra complexity? I'm concerned that the impedance mismatch here ends up looking like the most important aspect of the function, given the amount of code dedicated to it.

Reply:
Mmm, I had the same thought. Let me look at it again to see :-)

Reply:
Any more thoughts on this? We're nearly there :)

Reply:
I've changed this into a pattern match over three lines. I think it's more readable. However, if you would prefer, let me know and I'll just remove the entire Time line in both files.